src/index.js

```diff
@@ -130,7 +130,12 @@ export default async function microbundle(inputOptions) {
 	);
 
 	const targetDir = relative(cwd, dirname(options.output)) || '.';
-	const banner = blue(`Build "${options.name}" to ${targetDir}:`);
+	const sourceExist = options.input.length > 0;
+	const banner = sourceExist
+		? blue(`Build "${options.name}" to ${targetDir}:`)
+		: red(
+				`[!] Error: Can't resolve '${options.pkg.source}' in '${options.cwd}'`,
+		  );
```

Comment:
This might be confusing for folks who aren't using `"source"`. I am thinking maybe a more generic error message would avoid that:

```suggestion
		: red(`Error: No entry module found for "${options.name}"`);
```

Reply:
Can you provide me a example message I'll create a pull request